internal/server/disk/internal/types.go

```diff
@@ -0,0 +1,43 @@
+package internal
+
+type DiskIDList struct {
+	// List of node metrics.
+	IDs []string
+}
+
+type DiskLocation struct {
+	Adapter string
+	Bus     string
+	Target  string
+	LUNID   string
+}
+
+type ListDiskIDsRequest struct {
+}
+
+type ListDiskIDsResponse struct {
+	// Map of disk device IDs and SCSI IDs associated with each disk device
+	Disk_IDs map[string]*DiskIDList
```

Comment:
Is there a reason why we have to use _ ? Would it be possible to keep it consistent with the camelcase names ?